### PR TITLE
[master]fix sles dns not configured error

### PR DESCRIPTION
**Pull request by wgxoyun**
fix sles dns not configured error
Signed-off-by: wgxoyun <wgxoyun@cn.ibm.com>

### CODE REVIEW

**Comment by wgxoyun** [comment]
```
(venv3) [root@fba82 python-zvm-sdk]# tox -e pep8
pep8 develop-inst-noop: /root/wxycode/python-zvm-sdk
pep8 installed: attrs==20.1.0,certifi==2020.6.20,chardet==3.0.4,flake8==3.8.3,idna==2.10,importlib-metadata==1.7.0,importlib-resources==3.0.0,jsonschema==3.2.0,mccabe==0.6.1,netaddr==0.8.0,pycodestyle==2.6.0,pyflakes==2.2.0,PyJWT==1.7.1,pyrsistent==0.16.0,PyYAML==5.3.1,repoze.lru==0.7,requests==2.24.0,Routes==2.4.1,six==1.15.0,urllib3==1.25.10,WebOb==1.8.6,zipp==3.1.0,-e git+https://wgxoyun@github.com/wgxoyun/python-zvm-sdk.git@e03069106b9033f95e9f81b5d03170a03a017b78#egg=zVMCloudConnector
pep8 run-test-pre: PYTHONHASHSEED='3824004747'
pep8 run-test: commands[0] | flake8
pep8 run-test: commands[1] | flake8 /root/wxycode/python-zvm-sdk/scripts/sdkserver
__________________________________________________________________________ summary ___________________________________________________________________________
  pep8: commands succeeded
  congratulations :)
```

```
Ran 647 tests in 1.079s

OK (skipped=3)
py36 run-test: commands[1] | python -m unittest discover -v -s /root/wxycode/python-zvm-sdk/smtLayer/tests/unit
test_getDiskPoolSpace3390 (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace3390Unknown (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace3390UnknownWithFlag (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336 (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336Unknown (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336UnknownWithFlag (test_getHost.SMTGetHostTestCase) ... ok
test_getReservedMemSize (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_equal_size (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_gap_G (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_invalid_suffix (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_max_less_than_initial (test_makeVM.SMTMakeVMTestCase) ... ok
test_getVM_directory_py3 (test_vmUtils.SMTvmUtilsTestCase) ... ok

----------------------------------------------------------------------
Ran 12 tests in 0.004s

OK
__________________________________________________________________________ summary ___________________________________________________________________________
  py36: commands succeeded
  congratulations :)
```

File: zvmsdk/dist.py [hunk 1]
```diff
@@ -1499,7 +1505,24 @@ def _enable_network_interface(self, device, ip, broadcast):
 
 class sles15(sles12):
     """docstring for sles15"""
-    pass
+    def get_znetconfig_contents(self):
+        remove_route = 'rm -f %s/ifroute-eth*' % self._get_network_file_path()
+        replace_var = 'NETCONFIG_DNS_STATIC_SERVERS'
+        replace_file = '/etc/sysconfig/network/config'
+        dns_addrs = ''.join(self.dns_v4)
```
Comment (bjhuangr) [inline zvmsdk/dist.py:1512]: no space (or other separator) between multiple dns ?

**Review by bjhuangr — APPROVED** [review]
LGTM

**Review by bjhuangr — APPROVED** [review]

**Review by dongyanyang — APPROVED** [review]

**Comment by jichenjc** [comment]
@mfriesenegger this is the fix for DNS settings, can you help to check whether it's your desired fix or any further comments? Thanks

**Comment by mfriesenegger** [comment]
@jichenjc The fix works.  I will open a new issue if needed.  Thank you!

**Pull request merged**